news/4.33/platform.html

```diff
@@ -141,6 +141,38 @@ <h2>Preferences </h2>
     <h2>Themes and Styling </h2>
     </td>
   </tr>
+  <tr id="new-light-theme"> <!-- https://github.com/eclipse-platform/eclipse.platform.ui/issues/2114 -->
+      <td class="title">Preview Version of Improved Light Theme</td>
+      <td class="content">
+        <p>
+        The Eclipse default light theme has not been updated in years. It is not state 
+        of the art anymore. All major IDEs have shifted to a different light theme style.
+        When comparing the different IDEs it is easy to pick out the design decisions 
+        they all share: 
```

Comment:
May this paragraph should be formulated more positively and should should mention less prominently that the redesign is ('just') inspired by other IDEs?
For example
```suggestion
        After many years of stability we have reworked the light theme of Eclipse to follow more modern design principles, similar to other major IDEs:
```